build.xml

```diff
@@ -168,15 +168,17 @@
 		<property name="ant.gz.url" value="${ant.base.url}${ant.gz}"/>
 		<property name="ant.dir" value="build/apache-ant-${ant.version.}"/>
 		<property name="rhino.version.major" value="1"/>
+		<!-- March 20 2025 Rhino version not updated. Is used only within the Frank!Runner, not by the FF!.
+		     It seems that the next version, 1.8.0 is no longer free -->
```

Comment:
This comment is incorrect. It is still free but not available as zip download anymore

Reply:
Vandaag heb ik het gezien. Nu is deze comment weg.